docs/chatterino-api.service

```diff
@@ -4,14 +4,22 @@ DefaultDependencies=no
 
 [Service]
 User=billy
-ExecStart=/path/to/api/binary -l "127.0.0.1:1234" -b "https://domain.for.api"
+ExecStart=/path/to/api/binary"
 Restart=always
 RestartSec=5s
-Environment="CHATTERINO_API_DISCORD_TOKEN=XXXXXXXXXXXXXXX.XXXXX.XXXXXXXXXXXXXX"
+Environment="CHATTERINO_API_BASE_URL="
+Environment="CHATTERINO_API_BIND_ADDRESS=:1234"
+Environment="CHATTERINO_API_MAX_CONTENT_LENGTH=5242880"
+Environment="CHATTERINO_API_ENABLE_LILLIPUT=true"
+Environment="CHATTERINO_API_DISCORD_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
 Environment="CHATTERINO_API_CACHE_TWITCH_CLIENT_ID=XXXXXXXXXXXXXXXXXXXXXXXX"
-Environment="YOUTUBE_API_KEY=XXXXXXXXXXXXXXX"
+Environment="CHATTERINO_API_TWITCH_CLIENT_SECRET=XXXXXXXXXXXXXXXXXXXXXXXX"
+Environment="CHATTERINO_API_YOUTUBE_API_KEY=XXXXXXXXXXXXXXX"
 Environment="CHATTERINO_API_TWITTER_BEARER_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXX"
 Environment="CHATTERINO_API_IMGUR_CLIENT_ID=XXXXXXXXXXXXXXXXXXXXXXXXXXX"
+Environment="CHATTERINO_API_OEMBED_FACEBOOK_APP_ID=XXXXXXXXXXXXXXX"
+Environment="CHATTERINO_API_OEMBED_FACEBOOK_APP_SECRET=XXXXXXXXXXXXXXX"
+Environment="CHATTERINO_API_OEMBED_PROVIDERS_PATH=./providers.json"
```

Comment:
Should we comment these out as well by default?

Reply:
Sure, makes sense to do that to keep consistency.